desktop/utils/domain/auctions/urls.js

```diff
@@ -0,0 +1,19 @@
+const sd = require('sharify').data
+
+/** Get the live auction url with /login appended when a user is present
+ *  Uses sd.PREDICTION_URL or falls back to process.env in case sd is not fully initialized yet
+ * @param {String} id - auction id/slug
+ * @param {Object} [options]
+ * @param {Boolean} [options.isLoggedIn] - whether there is a user present
+ * @return {String} live auction url
+ */
+export const liveAuctionUrl = (id, options = {}) => {
+  const liveAuctionRoot = sd.PREDICTION_URL || process.env.PREDICTION_URL
```

Comment:
Ah, I see now... that race condition you describe where sharify data isn't loaded before the redirect seems strange. The loading of config data into sharify should happen at "compile time" [around here](https://github.com/artsy/force/blob/master/lib/setup.js#L27). Maybe we can screen hero to debug this problem.